test-setup/src/index.ts

```diff
@@ -60,8 +62,11 @@ async function startServer() {
       }
     }
     const sessionIds = await userDataSource.createLoggedInUsers(userIds);
+    const reviewerUserIds = userIds.slice(0, 100);
+    console.log(`trying to assign roles to users from test-setup`);
+    console.info(`the user ids going to be reviewers will be ${reviewerUserIds}`);
+    await userDataSource.assignRoleToUsers(reviewerUserIds, 'fapMember');
```

Comment:
This means every user created will be assigned reviewer role which we may not want for every test can we put it maybe behind a flag.